app/models/miq_retire_task.rb

```diff
@@ -11,7 +11,7 @@ def self.get_description(req_obj)
              req_obj.source.name
            end
 
-    "#{request_class::TASK_DESCRIPTION} for: #{name}"
+    N_("#{request_class::TASK_DESCRIPTION} for: #{name}")
```

Comment:
This seems problematic from an i18n standpoint as we shouldn't be using interpolation.  Instead I think this should be something like:

```suggestion
    _("%{request_description} for: %{request_source}") % {:request_description => _(request_class::TASK_DESCRIPTION), :request_source => name}
```

Then where the TASK_DESCRIPTIONs are defined, you wrap _those_ in `N_`